openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/CubeOpenshiftExtension.java

```diff
@@ -69,7 +70,7 @@ public void register(ExtensionBuilder builder) {
         if (Validate.classExists("org.jboss.arquillian.container.test.spi.client.deployment.AuxiliaryArchiveAppender")
             && doesNotContainStandaloneExtension()) {
             builder.service(AuxiliaryArchiveAppender.class, UtilsArchiveAppender.class);
-            builder.service(DeploymentScenarioGenerator.class, ExternalDeploymentScenarioGenerator.class);
+            builder.override(DeploymentScenarioGenerator.class, AnnotationDeploymentScenarioGenerator.class, ExternalDeploymentScenarioGenerator.class);
```

Comment:
I am not sure why are we overriding this as a part of this PR. wouldn't it be a good to add it when you are updating version of core in cube, so we can relate it in the future?

Reply:
No strong opinion about it. But as discussed with @lordofthejars, since it was a small change we thought of adding it as part of this PR since both the issues were related to the same tests.

Reply:
Can move it to https://github.com/arquillian/arquillian-cube/pull/994 or create a new PR for this? @dipak-pawar, what do you suggest?

Reply:
I would move it to #994, as we found it there.